clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/UbamTests.scala

```diff
@@ -506,14 +424,18 @@ trait UbamTests { self: BaseIntegrationSpec =>
     }
   }
 
-  def testMoveUbam(srcIsDest: Boolean = false): Future[Assertion] = {
+  def testMoveUbam(
+    srcIsDest: Boolean = false,
+    // TODO: Rip this param out when finished phasing out wgs command usage in other codebases
+    useOldCommand: Boolean = false
+  ): Future[Assertion] = {
     val barcode = s"barcode$randomId"
     val lane = 3
     val library = s"lib$randomId"
 
     val fileContents = s"$randomId --- I am a dummy ubam --- $randomId"
     val ubamName = s"$randomId${UbamExtensions.UbamExtension}"
-    val sourceDir = rootTestStorageDir / s"wgs-ubam/$barcode/$lane/$library/$ubamName/"
+    val sourceDir = rootTestStorageDir / s"ubam/$barcode/$lane/$library/$ubamName/"
```

Comment:
I know this is a test, but are there plans to move existing ubams to this new bucket?

Reply:
I don't know, but ultimately I don't think it matters in Clio. My understanding is that Clio, given the correct credentials, should be able to move things between arbitrary buckets

Reply:
This is a temp path for the fake ubams generated per-test, so it's safe to change. The stuff we stick in here gets deleted at the end of every test so there's nothing to migrate.

Reply:
👍